Shared/Layer.cs

```diff
@@ -0,0 +1,23 @@
+namespace ScintillaNET;
+
+/// <summary>
+/// 
```

Comment:
Perhaps this missing summary could be added. I can add it too after work, otherwise looks great.👍